cmd/zpool/zpool_main.c

```diff
@@ -9094,6 +9094,7 @@ zpool_do_status(int argc, char **argv)
 		if (count != 0 && --count == 0)
 			break;
 
+		(void) fflush(stdout);
```

Comment:
We should probably add this to all of the sub-commands which take an interval an can sleep.

Reply:
Dammit, this was supposed to be a casual drive-by patch. Fine! :laughing: